posthog/api/feature_flag.py

```diff
@@ -106,3 +109,9 @@ def get_queryset(self) -> QuerySet:
         if self.action == "list":
             queryset = queryset.filter(deleted=False)
         return queryset.order_by("-created_at")
+
+    @action(methods=["GET"], detail=False)
+    def user_status(self, request: request.Request, **kwargs):
```

Comment:
Also not a big deal, but `user_status` didn't seem to explain well what this endpoint does. Perhaps `for_user`? Unsure about the best semantics here.

Reply:
Renamed to "for_user", though I'm also not sure what's the best name/convention here.

Reply:
Yeah not a huge fan either, but probably not worth spending any more time on this.